internal/syncer/connector/bitbucket.go

```
@@ -0,0 +1,1014 @@
+			}
+			if err := sleepFor(ctx, delay); err != nil {
+				return nil, err
+			}
+			delay = bitbucketNextDelay(delay)
+			continue
+		}
+
+		if resp.StatusCode == http.StatusTooManyRequests {
+			retryAfter := bitbucketRetryAfter(resp)
+			_, _ = io.Copy(io.Discard, io.LimitReader(resp.Body, 4096))
+			resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)